File: solutions/uri/1066/1066.cpp
#include <cstdio>

int
    main() {
    int c = 5, x, p = 0, n = 0, e = 0, o = 0;

    while (c--) {
        scanf("%d", &x);

        if (x < 0) {
            n++;
        } else if (x > 0) {
            p++;
        }
        if (x % 2 == 0) {
            e++;
        } else {
            o++;
        }
    }

    printf("%d valor(es) par(es)\n", e);
    printf("%d valor(es) impar(es)\n", o);
    printf("%d valor(es) positivo(s)\n", p);
    printf("%d valor(es) negativo(s)\n", n);

    return 0;
}

File: solutions/beecrowd/1066/1066.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

p = n = e = o = 0

for _ in range(5):
    x = int(input())

    if x < 0:
        n += 1
    elif x > 0:
        p += 1
    if x % 2 == 0:
        e += 1
    else:
        o += 1

output(f'{e} valor(es) par(es)\n');
output(f'{o} valor(es) impar(es)\n');
output(f'{p} valor(es) positivo(s)\n');
output(f'{n} valor(es) negativo(s)\n');
